### ISSUE TITLE
Unexpected character escaping and sorting

### ISSUE THREAD

**Issue by achimmihca** [marhali/easy-i18n#10]
I have a properties (encoded in UTF-8) file like
```
# test_comment1=this is the first comment
! test_comment2=this is the second comment
test_helloWorld=Hello world!
test_equals=before=after
test_parameters=First name: {firstName}, second name: {secondName}
test_nonAscii=üäößêéèáà
test_quotes="All these outer and 'single quotes' and "double quotes" are kept"
test_spaceAtTheEndNotAroundEquals  =  Space around the equals sign is trimmed, but the ending space is kept: 
test_tab=Before tab\tafter tab
test_newline=First line\nAfter newline\rAfter carriage return
test_KeepSpace=\ <- white-space -> 
test_targetCities=\
        Detroit,\
        Chicago,\
        Los Angeles
test_backslash=\\Just two backslashes\\
```

When I use Easy-i18n, then
- The lines are reordered
    - I can't say how they are ordered. It is not alphabetically.
- Some characters are escaped.
    - for example, : becomes \:  and ! becomes \! and = becomes \=
- Everything changes, even when I only change a single translation

I did not expect this.

- Unneeded escaping of characters should not happen.
- Reordering translations would be nice as an optional feature. The order should then be alphabetically (or maybe configurable).
- Lines where the translation has not been changed should stay the same.
    - To implement this, the lines of the original file could be stored in order. This data structure is updated on translation change. On Save, this data structure is flushed back to the file.
    - Note that this is relevant for version control (to see which translations have been changed).

**Comment by marhali** [comment]
- Encoding is still not working as expected (includes JSON translator)
- PropertiesTranslator is based on Java's Properties class which does not care about key sorting

TODO:
- Write own Properties implementation which sorts by key alphabetically
- Enforce correct encoding

**Comment by marhali** [comment]
Should have been fixed now with latest release [v1.2.0](https://github.com/marhali/easy-i18n/releases/tag/v1.2.0).

There are still some characters which will be escaped but in plain form like: `test=yes=no => test=yes\=no`
But this is the same behaviour as in the standard implementation from Java's Properties class.
